WalletWasabi/Microservices/Binaries/osx64/Tor/tor

```diff
@@ -9,4 +9,4 @@ export DYLD_LIBRARY_PATH=.:$DYLD_LIBRARY_PATH
 # https://trac.torproject.org/projects/tor/ticket/10030
 cd "$(dirname "$0")"
 exec ./tor.real "$@"
-# Random Comment Dump to change the digest "je202kjnfoj2iuwkekjjreiejrd"
+# Random Comment Dump to change the digest "je202kjnfoj2iuwkekjjreiejre"
```

Comment:
There is a change in the permissions as well 755=>644
I am not sure if that is a problem, please verify it. Did you do the chmod commands described here? 
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Microservices/Binaries/README.md

Reply:
Theoretical question (no practical sense, except to clarify): The modification of the digest is not required anymore. Old clients copied Tor executables to that data folder and ran the client from there. Now we are running directly from the binary folder (like program files on windows). The digest was used to compare if an update is required, thus comparing the tor executables in the data folder and in the binary folder.
If this is true we should update the documentation and stop modifying this.

Reply:
Fixed the permissions